script/tasks/diamondSyncWhitelistedAddresses.sh

```
@@ -96,73 +96,73 @@
-        echo "[info] Trying to add missing DEXs now - attempt ${ATTEMPTS} (max attempts: $MAX_ATTEMPTS_PER_SCRIPT_EXECUTION) "
+        echo "[info] Trying to add missing whitelisted addresses now - attempt ${ATTEMPTS} (max attempts: $MAX_ATTEMPTS_PER_SCRIPT_EXECUTION) "
         doNotContinueUnlessGasIsBelowThreshold "$NETWORK"
 
-        cast send "$DIAMOND_ADDRESS" "batchAddDex(address[])" "${PARAMS[@]}" --rpc-url "$RPC_URL" --private-key $(getPrivateKey "$NETWORK" "$ENVIRONMENT") --legacy >/dev/null
+        cast send "$DIAMOND_ADDRESS" "batchAddToWhitelist(address[])" "${PARAMS[@]}" --rpc-url "$RPC_URL" --private-key $(getPrivateKey "$NETWORK" "$ENVIRONMENT") --legacy >/dev/null
 
         sleep 5 # Wait for confirmation
 
         # Check on-chain state after transaction
-        DEXS_UPDATED=($(getApprovedDEXs))
+        WHITELISTED_ADDRESSES_UPDATED=($(getWhitelistedAddresses))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)